src/Xamarin.Android.Build.Tasks/Utilities/ManagedResourceParser.cs

```diff
@@ -646,7 +662,7 @@ void ProcessStyleable (XmlReader reader)
 				CodeArrayCreateExpression c = field.InitExpression as CodeArrayCreateExpression;
 				if (c == null)
 					return;
-				
+				Log.LogDebugMessage ($"Adding array Mapping {field.Name} {fields.Count}");
```

Comment:
Maybe you meant to take these `LogDebugMessage` out, and they were just left for debugging?